setup/src/Magento/Setup/Model/Installer.php

```diff
@@ -925,7 +925,7 @@ private function throwExceptionForNotWritablePaths(array $paths)
      */
     private function handleDBSchemaData($setup, $type, array $request)
     {
-        if (!(($type === 'schema') || ($type === 'data'))) {
+        if (!($type === 'schema') || ($type === 'data')) {
```

Comment:
@DanielRuf Looks like this changes the existing logic. Can you please review it again? Note the original code is checking IF NOT `(($type === 'schema') || ($type === 'data'))` whereas the new code is checking IF NOT `($type === 'schema')` OR IF TRUE `($type === 'data')`.

Reply:
You are right, I will fix this.

Reply:
Should be ok now.